tdrs-backend/tdpservice/stts/management/commands/data/tribes.csv

```diff
@@ -0,0 +1,117 @@
+Name,Region,Code,State
```

Comment:
A question about this Tribal data — is this data final & correct data for Tribes, or is it sample data? Should we document the state of this data — finalized & approved by TANF, sample data, mostly right but waiting on final confirmation, etc.?